Library/Homebrew/formula_installer.rb

```diff
@@ -55,6 +56,13 @@ def initialize(formula)
     @pour_failed   = false
   end
 
+  # called by install/reinstall/upgrade when no build tools are available
+  def self.check_build_flags
```

Comment:
`check_build_flags` doesn't seem perfectly descriptive.  It's specifically checking that there are no non-default flags, right?